build-tests/x86/fedora/test-image-live-disk/appliance.kiwi

```diff
@@ -30,7 +30,11 @@
         </type>
     </preferences>
     <preferences profiles="Virtual">
-        <type image="oem" filesystem="ext4" kernelcmdline="console=ttyS0" firmware="uefi" format="qcow2">
+        <type image="oem" filesystem="btrfs" kernelcmdline="console=ttyS0 rootflags=subvol=root" firmware="uefi" format="qcow2" btrfs_root_is_subvolume="true" btrfs_set_default_volume="false" rootfs_label="fedora" bootpartition="true" bootfilesystem="ext4">
```

Comment:
`rootflags=subvol=root` should probably be automatic as a consequence of `btrfs_root_is_subvolume="true"`. Requiring that it be specified manually is probably going bite people down the road.

Reply:
Hmm, but it feels distro specific. Setting ```rootflags=subvol=root``` on suse as an example would be useless information. Or is this somewhat standardized ?

Reply:
I'd like to prevent implicit magic unless it would be generally valid.

Reply:
That is absolutely standard. You would be propagating the `subvol=<rootsubvolname>` thing you already are doing for fstab to the kernel command line, just with the prefix `rootflags=`. By definition, this is not implicit magic like `btrfs subvolume set-default`, since you're explicitly telling it where to look.

Reply:
Basically, if you don't do `btrfs subvolume set-default` to reorient the filesystem for the root tree, you need to tell the kernel where the root tree is.

Reply:
ok that makes sense to me. I will adapt and it should also not hurt.

Reply:
added a commit to solve this, we need to check on the integration test results though